src/Spectre.Service.Tests/Loaders/DatasetLoaderTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿/*
+ * DatasetLoaderTest.cs
+ * Tests for dataset loader class.
+ *
+   Copyright 2017 Dariusz Kuchta
+   
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+   
+   http://www.apache.org/licenses/LICENSE-2.0
+   
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.IO.Abstractions.TestingHelpers;
+using System.Linq;
+using Ninject;
+using Ninject.Parameters;
+using NUnit.Framework;
+using Spectre.Service.Configuration;
+using Spectre.Service.Loaders;
+
+namespace Spectre.Service.Tests.Loaders
+{
+    [TestFixture]
+    public class DatasetLoaderTest
+    {
+        private DataRootConfig _rootConfig;
+        private DatasetLoader _datasetLoader;
+        private MockFileSystem _mockFileSystem;
+
+        private readonly string _rootDir = @"C:\spectre_data";
+        private readonly string _localDir = @"\local";
+        private readonly string _remoteDir = @"\remote";
+
+        private readonly string _fileDir = TestContext.CurrentContext.TestDirectory
```

Comment:
Try using string interpolation for evaluating string expressions. It condenses the code and calls string format underneath, which makes it more localization safe than just adding strings.

Reply:
Or you could just use Path.Combine here :)